elements/xml_form_elements.module

```diff
@@ -303,11 +303,35 @@ function xml_form_elements_form_element_tabs_ajax_add(FormElement $element, arra
     if ($element->controls['#type'] == 'tags') {
       foreach ($element->children as $child) {
         $child->default_value = $child->getOriginalDefaultValue();
-        if (empty($child->default_value)) {
+
+        // We should orphan object only if they have no default AND it is not
+        // the last child.  We must retain one, otherwise we'll get AJAX an
+        // AJAX error when adding an element after adding another tab.
+        $no_default = empty($child->default_value);
+        $is_not_last = count($element->children) > 1;
+
+        if ($no_default && $is_not_last) {
           $child->orphan();
         }
       }
+
+      // In order to provide functionality identical to when the tab is first
+      // created, we need to dedeupe based on the first element's default
+      // value.  Otherwise we can wind up with multiples of the same default
+      // value.
+      if (count($element->children > 1)) {
+        $first = array_shift($element->children);
+
+        foreach ($element->children as $child) {
+          if (strcmp($child->default_value, $first->default_value) == 0) {
```

Comment:
I'm not sure this is a good assumption, that the same value will not be present twice... Could be legitimate (though likely rare... can't think of an exact use-case off the top of my head). But yeah: Created a form which does it: https://gist.github.com/adam-vessey/935b8180dc9588a86e35

When hitting the button for a new tabpanel, one of the "default_0" dealios goes away in the clone.